packages/cli/test/simulation/fourNodesForks.test.ts

```diff
@@ -0,0 +1,86 @@
+import chai, {expect} from "chai";
+import chaiAsPromised from "chai-as-promised";
+import {routes} from "@lodestar/api/beacon";
+import {logFilesDir, SimulationEnvironment, FAR_FUTURE_EPOCH} from "../utils/simulation/index.js";
+
+chai.use(chaiAsPromised);
+
+describe.skip("fourNodesForks", function () {
+  this.timeout("5m");
```

Comment:
How is this 5m figure computed? Does it provide enough time for tests to complete? You should be able to compute the required timeout

Reply:
Yes it works fine with 5m. We can compute the time but that would be a best case scenario, it something is broken and tests are not passing then it could timeout before assertions and we will not be able to see which assertions are failing.